dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -1231,3 +1231,71 @@ def test_profile_merge_bin_edges_indices(self):
         profile_2.update(data_2)
 
         profile_1 + profile_2
+
+    def test_invalid_values_skew_kurt(self):
+        data = pd.Series(['-inf', 'inf'])
+        profiler = FloatColumn(data.name)
+
+        with warnings.catch_warnings(record=True) as w:
+            warnings.simplefilter("always")
+
+            profiler.update(data)
+            # Verify values are NaN
+            self.assertTrue(np.isnan(profiler._biased_skewness))
+            self.assertTrue(np.isnan(profiler._biased_kurtosis))
+            # Verify warning was raised properly
+            self.assertEqual(2, len(w))
```

Comment:
should variance also have a warning considering it also would return nan, yes?

Reply:
I agree, however this wasn't in the AC of the original task. I was going to bring this up with you, just never had the chance.

Reply:
It's also a bit different, since variance relies on np.var, not one of our independently written functions

Reply:
True, but we would expect the same occurrence of the issue, no?